apps/testingaccessibility/src/pages/api/certificate/index.tsx

```diff
@@ -0,0 +1,148 @@
+import {ImageResponse} from '@vercel/og'
```

Comment:
I refactored the generation logic to use vercel/og™ and removed external service connection